docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/openshift.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Configure the OpenShift Cloud Provider
+headerTitle: Configure the OpenShift Cloud Provider
+linkTitle: Configure the cloud provider
+description: Configure the OpenShift cloud provider
+aliases:
+  - /latest/deploy/enterprise-edition/configure-cloud-providers/openshift
+menu:
+  latest:
+    identifier: set-up-cloud-provider-5-openshift
+    parent: configure-yugabyte-platform
+    weight: 20
+isTocNested: false
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li>
+    <a href="/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws" class="nav-link">
+      <i class="fab fa-aws"></i>
+      AWS
+    </a>
+  </li>
+
+  <li>
+    <a href="/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/gcp" class="nav-link">
+      <i class="fab fa-google" aria-hidden="true"></i>
+      GCP
+    </a>
+  </li>
+
+  <li>
+    <a href="/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/azure" class="nav-link">
+      <i class="icon-azure" aria-hidden="true"></i>
+      &nbsp;&nbsp; Azure
+    </a>
+  </li>
+
+  <li>
+    <a href="/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes" class="nav-link">
+      <i class="fas fa-cubes" aria-hidden="true"></i>
+      Kubernetes
+    </a>
+  </li>
+
+  <li>
+    <a href="/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/vmware-tanzu" class="nav-link">
+      <i class="fas fa-cubes" aria-hidden="true"></i>
+      VMware Tanzu
+    </a>
+  </li>
+
+<li>
+    <a href="/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/openshift" class="nav-link active">
+      <i class="fas fa-cubes" aria-hidden="true"></i>OpenShift</a>
+  </li>
+
+  <li>
+    <a href="/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises" class="nav-link">
+      <i class="fas fa-building"></i>
+      On-premises
+    </a>
+  </li>
+
+</ul>
+
+This document describes how to configure OpenShift for YugabyteDB universes using Yugabyte Platform. If no cloud providers are configured via the Yugabyte Platform console, the main Dashboard page requests to configure at least one provider.
+
+![Configure Cloud Provider](/images/ee/configure-cloud-provider.png)
+
+## Creating a YugabyteDB Universe via Yugabyte Platform
```

Comment:
Heading seems a bit misleading. We are not creating a YugabyteDB universe yet

Reply:
Changed the document structure.